tofu/tests/tests01_geom/tests03_core.py

```diff
@@ -421,12 +421,13 @@ def test15_get_sampleV(self):
                             pts0, ind0 = out[0], out[2]
                         except Exception as err:
                             msg = (str(err) +
-                                   "\nFailed for {0}_{1}_{2}\n".format(typ, c, n)
+                                   "\nFailed for {0}_{1}_{2}\n".format(typ,
+                                                                       c, n)
```

Comment:
Here for example the updated version, but black isnt showing them :shrug:

Reply:
ok, fine for me , thanks !